modules/eventing/pages/eventing-handler-keepLastN_661.adoc

```diff
@@ -0,0 +1,240 @@
+= Function: Keep the Last N User Items 6.6.1
+:page-edition: Enterprise Edition
+:tabs:
+
+*Goal*: Keep the last N user notifications seen related to a user ID (these could be any documents) [.status]#Couchbase Server 6.6.1# 
+
+* This function *keepLastN_661* demonstrates how to keep a user record with the last N activities.
+* Requires a metadata bucket and a source bucket.
+* Requires an alias of src_bkt to the source bucket in read+write mode.
+* Will operate on any mutation with a key starting with "nu:" of the form "nu:#:#".
+* The key "nu:#:#" contains two numbers. The first # is an increasing notification number, the second # is the user ID.
+* Anytime we insert a new record we want to remove the earliest notification record for the user so we only have at most N records for each user.
+We assume that nid always increases across time as such we ignore duplicates.
+* For our test we will keep just the three (3) most recent notifications per user ID.
+
+[{tabs}] 
+====
+keepLastN_661::
++
+--
+Two variants of this function are available a 6.6 version that implements userspace CAS and a 6.6.1 version (*this Function*) which uses true CAS
+
+* xref:eventing-handler-keepLastN.adoc[keepLastN (userspace CAS)]
+* <<keepLastN_661,keepLastN_661 (true CAS)>>
+
+[#keepLastN_661]
```

Comment:
rename to [#advanced_keepLastN] so the URL stay the same for future releases as well

Reply:
There are two versions (one will work up through 6.6.0 and the other will work starting from 6.6.1 on)
eventing-handler-keepLastN.adoc
eventing-handler-keepLastN_661.adoc
I have no issues with renaming - but we still need the two versions since this is a 6.6.0 document set.